app-frontend/src/app/components/map/mapContainer/mapContainer.controller.js

```diff
@@ -2,14 +2,16 @@ import Map from 'es6-map';
 /* globals BUILDCONFIG */
 
 export default class MapContainerController {
-    constructor($log, $document, $element, $scope, $timeout, mapService) {
+    constructor($log, $document, $element, $scope, $timeout, mapService,
```

Comment:
Kind of a nitpick, but I've been trying to organize injections in controllers like this if it runs over a line:
```javascript
constructor(
  $inject, $inject, $inject, $inject ....,
  service, service, service, service
) { ... }
```